experimental/generation/generator/packages/library/src/mergeAssets.ts

```diff
@@ -747,3 +707,30 @@ async function parseSchemas(schemaName: string, oldPath: string, newPath: string
     await copySingleFile(newPath, mergedPath, schemaName + '.json', newFileList, feedback)
     return {oldPropertySet, newPropertySet}
 }
+
+/**
+ * @description: Get the synonyms of the enum entity.
+ * @param schemaName Name of the .schema file.
+ * @param newPath Path to the folder of the new asset.
+ */
+async function getSynonyms(schemaName: string, newPath: string): Promise<Map<string, Set<string>>> {
+    let template = await fs.readFile(ppath.join(newPath, schemaName + '.json'), 'utf8')
+    let newObj = JSON.parse(template)
+    let propertyValueSynonyms = new Map<string, Set<string>>()
+
+    for (let property in newObj['properties']) {
+        let examples = newObj['properties'][property]['$examples']
+        if (examples !== undefined && examples[''][`${property}Value`] !== undefined) {
```

Comment:
There are 4 possible sources of $examples in order.
1. property/$examples/locale/entity
2. property/$examples/""/entity
3. $examples/locale/entity
4. $examples/""/entity

Reply:
This is still wrong.  You need to be checking the specific locale you are generating, i.e. for foo.en-us.lu you look for a locale in the property then the default locale, then the same in the global $examples.

---
In reply to: [587057402](https://github.com/microsoft/BotBuilder-Samples/pull/3104#discussion_r587057402) [](ancestors = 587057402)